### PR TITLE
{bio} [dummy] PLINK-1.90b (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
we have existing `PLINK` easyconfigs that use `MakeCp` + a proper toolchain, any particular reason you're going for `dummy` here?

Those easyconfigs also don't have the versionless-sources issue...

File: easybuild/easyconfigs/p/PLINK/PLINK-1.90b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+easyblock = "Tarball"
+
+name = 'PLINK'
+version = '1.90b'
+
+homepage = 'https://www.cog-genomics.org/plink2'
+description = "Whole-genome association analysis toolset"
+
+toolchain = {'name': 'dummy', 'version': ''}
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.90b.eb]: aiming for what?

Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.90b.eb]: matter little, imho?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fddff80f225ea0f6b98c for a full test report.

**Comment by boegel** [comment]
@pescobar: trouble in paradise?

**Comment by verdurin** [comment]
I've uploaded my own version at #3966 

The `prettify` binary doesn't work because the linking is broken, but the main binary appears to work.

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel it is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**